Graph Explorer- Updates (#915)
graphprotocol/docs: website/src/pages/en/subgraphs/explorer.mdx

title: Graph Explorer
---

Use [Graph Explorer](https://thegraph.com/explorer) and take full advantage of its core features.

## Overview

This guide explains how to use [Graph Explorer](https://thegraph.com/explorer) to quickly discover and interact with Subgraphs on The Graph Network, delegate GRT, view participant metrics, and analyze network performance.

> When you visit Graph Explorer, you can also access the link to [explore Substreams](https://substreams.dev/).

## Prerequisites

> - A wallet (e.g., MetaMask) connected to [Graph Explorer](https://thegraph.com/explorer).

Author: ```suggestion
- To perform actions, a need wallet (e.g., MetaMask) connected to [Graph Explorer](https://thegraph.com/explorer).
```

> - To perform actions, a need wallet (e.g., MetaMask) connected to [Graph Explorer](https://thegraph.com/explorer).

Reviewer: I think there's a typo here ("a need wallet"?)

Reviewer: Ah, and the "a" is in the wrong place hehe.
```suggestion
- To perform actions, you need a wallet (e.g., MetaMask) connected to [Graph Explorer](https://thegraph.com/explorer).
```

>   > Make sure your wallet is connected to the correct network (e.g., Arbitrum). Features and data shown are network specific.

Reviewer 2: Just as a note, users can view Graph Explorer without connecting, or even having a wallet. It is only needed at the point that they want to perform an action (like delegate, signal, etc)

- GRT tokens if you plan to delegate or curate.

> - Basic knowledge of Subgraphs

Reviewer: Maybe this should link to a page that explains what Subgraphs are, like [this one](https://thegraph.com/docs/en/subgraphs/developing/subgraphs/)?

## Navigating Graph Explorer

### Step 1. Explore Subgraphs

> For additional support, you can watch the [Graph Explorer video guide](/subgraphs/explorer/#video-guide).

> Go to the Subgraphs tab in [Graph Explorer](https://thegraph.com/explorer).

Reviewer: ```suggestion
Go to the Subgraphs page in [Graph Explorer](https://thegraph.com/explorer).
```

- If you've deployed and published your Subgraph in Subgraph Studio, you can view it here.
- Search all published Subgraphs and filter them by indexed network, specific categories (such as DeFI, NFTs, and DAOs), and **most queried, most curated, recently created, and recently updated**.

> ![Graph Explorer Subgraphs tab](/img/explorer-subgraphs-tab.png)

Reviewer: ```suggestion
![Graph Explorer Subgraphs page](/img/explorer-subgraphs-tab.png)
```

Reviewer: (the file should probably also be renamed, but... no big deal)

- For example, you can enter the L2GNS contract on Artbitrum (`0xec9A7fb6CbC2E41926127929c2dcE6e9c5D33Bec`) and this returns all Subgraphs indexing that contract:

![Search by Indexing Contract](/img/explorer-search.png)

> Looking for indexing contracts? Check out [this Subgraph](https://thegraph.com/explorer/subgraphs/FMTUN6d7sY2bLnAmNEPJTqiU3iuQht6ZXurpBh71wbWR?view=About&chain=arbitrum-one) which indexes contract addresses listed in its manifest. It shows all current deployments indexing those contracts on Arbitrum One, along with the signal allocated to each.

You can click into any Subgraph, to:

- Test queries in the playground and be able to leverage network details to make informed decisions.
- Signal GRT on your own Subgraph or the Subgraphs of others to make Indexers aware of its importance and quality.
  > This is critical because signaling on a Subgraph incentivizes it to be indexed, meaning it'll eventually surface on the network to serve queries.

![Subgraph Details](/img/explorer-subgraph-details.png)

On each Subgraph’s dedicated page, you can do the following:

- Query Subgraphs via GraphQL
- View Subgraph ID, current deployment ID, Query URL, and other metadata

> - Signal/Un-signal on Subgraphs

Reviewer: I don't think we use a hyphen in the UI:
```suggestion
- Signal/unsignal on Subgraphs
```

- Test Subgraphs in the playground
- View the Indexers that are indexing on a certain Subgraph
- Subgraph stats (allocations, Curators, etc)
- View query fees and charts
- Change versions to explore past iterations of the Subgraph
- View entity types
- View Subgraph activity

> ![Signal/Un-signal on Subgraph](/img/explorer-signal.png)

Reviewer: ```suggestion
![Signal/unsignal on a Subgraph](/img/explorer-signal.png)
```

### Step 2. Delegate GRT

> Go to the [Delegate](https://thegraph.com/explorer/delegate?chain=arbitrum-one) tab to learn how to delegate, get GRT, and choose and Indexer.

Reviewer: ```suggestion
Go to the [Delegate](https://thegraph.com/explorer/delegate?chain=arbitrum-one) page to learn how to delegate, get GRT, and choose an Indexer.
```

Here, you can:

> - Compare Indexers by most query fees earned and highest estimated APR
> - Use the built-in ROI calculator or search by Indexer name or address.
> - Click **"Delegate"** next to an Indexer to stake your GRT

Reviewer: ```suggestion
- Compare Indexers by most query fees earned and highest estimated APR.
- Use the built-in ROI calculator or search by Indexer name or address.
- Click **"Delegate"** next to an Indexer to stake your GRT.
```

### Step 3. Monitor Participants in the Network

> Go to the [Participants](https://thegraph.com/explorer/participants?chain=arbitrum-one) tab to view:

Reviewer: ```suggestion
Go to the [Participants](https://thegraph.com/explorer/participants?chain=arbitrum-one) page to view:
```

- Available Delegation Capacity: The amount of delegated stake the Indexers can still receive before they become over-delegated.
- Max Delegation Capacity: The maximum amount of delegated stake the Indexer can productively accept. An excess delegated stake cannot be used for allocations or rewards calculations.
- Query Fees: This is the total fees that end users have paid for queries from an Indexer over all time.
- Indexer Rewards: This is the total indexer rewards earned by the Indexer and their Delegators over all time. Indexer rewards are paid through GRT issuance.

Indexers can earn both query fees and indexing rewards. Functionally, this happens when network participants delegate GRT to an Indexer. This enables Indexers to receive query fees and rewards depending on their Indexer parameters.

- Indexing parameters can be set by clicking on the right-hand side of the table or by going into an Indexer’s profile and clicking the “Delegate” button.

To learn more about how to become an Indexer, you can take a look at the [official documentation](/indexing/overview/) or [The Graph Academy Indexer guides.](https://thegraph.academy/delegators/choosing-indexers/)

> ![Indexing Details Pane](/img/Indexing-Details-Pane.png)

Reviewer 3: ```suggestion
```

If you want to learn more about how to become a Delegator, check out the [official documentation](/resources/roles/delegating/delegating/) or [The Graph Academy](https://docs.thegraph.academy/official-docs/delegator/choosing-indexers).

### Step 4. Analyze Network Performance

> On the [Network](https://thegraph.com/explorer/network?chain=arbitrum-one) tab, you can see global KPIs and have the ability to switch to a per-epoch basis and analyze network metrics in more detail. These details will give you a sense of how the network is performing over time.

Reviewer: ```suggestion
On the [Network](https://thegraph.com/explorer/network?chain=arbitrum-one) page, you can see global KPIs and have the ability to switch to a per-epoch basis and analyze network metrics in more detail. These details will give you a sense of how the network is performing over time.
```

  - The distributing epochs are the epochs in which the state channels for the epochs are being settled and Indexers can claim their query fee rebates.
  - The finalized epochs are the epochs that have no query fee rebates left to claim by the Indexers.

![Epochs](/img/explorer-epochs.png)

## Access and Manage Your User Profile

### Step 1. Access Your Profile

> - Click your wallet address in the top right corner
> - Your wallet acts as your user profile
> - In your profile dashboard, you can view and interact with several useful tabs

Reviewer: ```suggestion
- Click your wallet address in the top right corner.
- Your wallet acts as your user profile.
- In your profile dashboard, you can view and interact with several useful tabs.
```

### Step 2. Explore the Tabs

#### Profile Overview

In this section, you can view the following:

- Your activity

> - Your profile information: total query fees, total shares value, owned stake. stake delegating

Reviewer: ```suggestion
- Your profile information: total query fees, total shares value, owned stake, stake delegating
```

![Explorer Image 10](/img/Profile-Overview.png)

#### Subgraphs Tab

The Subgraphs tab displays all your published Subgraphs.

> Subgraphs deployed with the CLI for testing purposes will not show up here. Subgraphs will only show up when they are published to the decentralized network.

![Explorer Image 11](/img/Subgraphs-Overview.png)

#### Indexing Tab

> > If you haven't indexed, you see links to stake to index Subgraphs and browse Subgraphs on Graph Explorer.

Reviewer: ```suggestion
> If you haven't indexed, you will see links to stake to index Subgraphs and browse Subgraphs on Graph Explorer.
```

The Indexing tab displays a table where you can review active and historical allocations to Subgraphs.

Track your Indexer performance with visual charts and key metrics, including:

- Delegated Stake - Stake from Delegators that can be allocated by you but cannot be slashed
- Total Query Fees - Cumulative fees from served queries
- Indexer Rewards (in GRT) - Total rewards earned
- Fee Cut & Rewards Cut - the % of query fee rebates and Indexer rewards you'll keep when you split with Delegators

> - Owned Stake - your deposited stake, which could be slashed for malicious or incorrect behavior

Reviewer: Should these hyphens also be replaced with `:` like you did above (under `#### Indexers` > `Specifics`)?

Reviewer: Also, should these list items end with periods?

> ![Explorer Image 12](/img/Indexer-Stats.png)

Reviewer: This could use a better alt text :P

#### Delegating Tab

> > To learn more about the benefits of delegating, check out [delegating](/resources/roles/delegating/delegating/)

Reviewer: ```suggestion
> To learn more about the benefits of delegating, check out [delegating](/resources/roles/delegating/delegating/).
```

The Delegators tab displays your active and historical delegations, along with the metrics for the Indexers you've delegated to.

Top Section:

- View delegation and rewards-only charts
- Track key metrics:
- Total delegation rewards
- Unrealized rewards
- Realized Rewards

> 

Reviewer: Should these be indented (are they the "key metrics" mentioned in the previous bullet point)?

Bottom Section:

> - Explore a table of your Indexer delegations, including reward cuts, cooldowns, and more
> - Use the buttons on the right side of the table to manage your delegation - delegate more, undelegate, or withdraw it after the thawing period

Reviewer: ```suggestion
- Explore a table of your Indexer delegations, including reward cuts, cooldowns, and more.
- Use the buttons on the right side of the table to manage your delegation - delegate more, undelegate, or withdraw it after the thawing period.
```

> > This table is horizontal scrollable, so scroll right to see delegation status: delegating, undelegating, or withdrawable.

Reviewer: ```suggestion
> This table is horizontally scrollable, so scroll right to see the delegation status: delegating, undelegating, or withdrawable.
```
